cardinal/world.go

```diff
@@ -163,6 +158,35 @@ func setLogLevel(levelStr string) error {
 	return nil
 }
 
+func applyProductionOptions(
+	cfg WorldConfig,
+	ecsOptions *[]ecs.Option,
+	serverOptions *[]server.Option,
+) error {
+	log.Logger.Info().Msg("Starting a new Cardinal world in production mode")
+	if cfg.RedisPassword == DefaultRedisPassword {
+		return eris.New("redis password is required in production")
```

Comment:
It would be helpful to include the environment variable with these error messages. e.g.

"in production, redis password must be set via the REDIS_PASSWORD environment variable"

Same with CardinalNamespace and BaseShard addresses below.

Reply:
[use env var keys in error message](https://github.com/Argus-Labs/world-engine/pull/511/commits/8e34f3b94295e11fe5cf1dfdc8168db46227ed5b)